### DISCUSSION TITLE
How to make acap app run at startup on power-up?

### DISCUSSION THREAD

**Discussion by AmarRobo** [AxisCommunications/discussions#180] [ACAP]
I am trying the object detection app, the app gets turned off on restarting the camera. The axis apps like object analytics or video motion detection remains running while the object detection app does not. Is there any option to make the object detection keep on running ?

**Comment by mattias-kindborg-at-work** [comment]
Are we talking about one of the examples in the [CV SDK repository](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples) or in the [Native SDK repository](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples)? If *"yes"* I would advice you to ask the question in that specific repository.

**Reply by AmarRobo** [reply]
Ok. Thanks
